spec/requests/v0/health_care_applications_spec.rb

```diff
@@ -15,6 +15,7 @@
     let(:current_user) { build(:ch33_dd_user) }
 
     before do
+      Flipper.disable(:hca_disable_bgs_service)
```

Comment:
Toggles are enabled by default in specs, so disabling it here to keep the specs consistent.